get_github_coderabbit_review/coderabbit_fetcher/ai/sentiment_analyzer.py

```
@@ -0,0 +1,704 @@
+                    tone=ToneType(data.get("tone", "neutral")),
+                    constructiveness=float(data.get("constructiveness", 0.5)),
+                    politeness=float(data.get("politeness", 0.5)),
+                    clarity=float(data.get("clarity", 0.5)),
+                    reasoning=data.get("reasoning", ""),
+                    keywords=data.get("keywords", []),
+                )
+
+            # Fallback to text parsing
+            return self._parse_text_response(response.content, text)
+
+        except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)